proposed/2021/NuGet.orgTFMs.md

```diff
@@ -0,0 +1,96 @@
+
+## Explanation
+
+### Functional Explanation
+
+#### Context and Scope 
+When a developer accesses the details page for a NuGet package, they will see information on target frameworks in a dropdown menu. The framework information will be surfaced from the package. In cases where versions are not specified (such as .net) or other unexpected scenarios, ‘Version Not Specified’ or similar messages will be surfaced. 
+
+##### Minimal Requirements 
+- List of Frameworks on NuGet.org
+
+##### Non-Requirements 
+- List of Platforms on NuGet.org
+  - Users interviewed did not express need for supported platform information 
+  - Cannot be surfaced directly from package data and would require additional layer of soft calculation from engineering (no clear mapping scheme currently exists between target frameworks and supported platforms) 
+  - Can be revisited in future 
+- Filters for Frameworks on NuGet.org
+    - Users interviewed were ambivalent on filters, requires further investigation. Can be revisited in future 
+- Filters for Platforms on NuGet.org
+  - Users interviewed were ambivalent on filters, requires further investigation. Can be revisited in future 
+
+Although Target Frameworks & Supported Platforms are relatively similar in general contexts and often have overlap, we must clearly define each of these items based on their differences. Today we live in an ecosystem where Target Frameworks can represent both a supported platform, or a subset of supported platforms. 
+
+**Target Framework** – A target framework is a specification of the APIs available to an application or library. 
+
+Example: .NET Framework, .NET Core, .NET Standard, .NET 5 are target frameworks. 
+
+**Supported Platform** – A supported platform is a target that the APIs can be ran on. 
+
+Example: iOS, Android, Windows are platforms. 
+
+### Solution 
+
+- A .NET developer can view Frameworks of a package on NuGet.org: 
+![](../../meta/resources/NuGet.orgTFMs/PackageDetailsWithTFMs.png)
+
+### Goals and Non-Goals 
+
+#### Goals 
+
+Make it clear to developers whether a package can be installed or updated for their project or solution based on a target framework. 
+
+#### Non-Goals 
+
+Over-complicate what it means to be a compatible package. One target framework + one target platform = compatible package. 
+
+### Design 
+
+#### Target Framework List 
+
+- .NET Framework (net) 
+- Xamarin/Mono (monoandroid, monotouch, xamarinios, monomac, xamarinmac, xamarinwatchos, xamarintvos) 
+- .NET Core (netcoreapp) 
+- .NET Standard (netstandard) 
+- .NET 5 (net5.0) 
+
+### Technical Explanation
+
+Packages whose layout follow our [guidelines](https://docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks) will be examined when uploaded and validated, prior to listing. Target Frameworks will be determined and displayed here.
+
+Note that packages which do not follow the guidelines may have inconsistent results.
+
+### Unresolved questions
```

Comment:
I think `build/<packageid>.[props|targets]` poses a problem, because from NuGet's point of view this package is compatible with all TFMs, .NET, native/c++, or anything else. For packages like Nerdbank.GitVersioning, or SourceLink, this makes sense.  But many .NET packages incorrectly use this, rather than `build\<tfm>\<packageid>.[props|targets]`. I think it's worthwhile pointing this out and explaining what our plan is for these packages.

Reply:
I've moved this into unresolved questions, @zivkan.